contracts/PetPark.sol

```diff
@@ -1,7 +1,86 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
-
 contract PetPark {
+    address public owner;
+
+    enum Pets {
+        None,
+        Fish,
+        Cat,
+        Dog,
+        Rabbit,
+        Parrot
+    }
+    enum Gender {
+        Male,
+        Female
+    }
+    struct Person {
+        Gender gender;
+        uint256 age;
+        Pets animal;
+        bool isExists;
+        bool isBorrowed;
```

Comment:
Is this field required? Can't you check using None animal type? Storage in solidity is very expensive hence we should reuse already declared states as much as possible